Src/AutoMoq/AutoMoq.csproj

```diff
@@ -12,16 +12,24 @@
     <PackageId>AutoFixture.AutoMoq</PackageId>
     <Title>AutoFixture with Auto Mocking using Moq</Title>
     <Description>This extension turns AutoFixture into an Auto-Mocking Container. The mock instances are created by Moq. To use it, add the AutoMoqCustomization to your Fixture instance. Read more at http://blog.ploeh.dk/2010/08/19/AutoFixtureAsAnAutomockingContainer.aspx</Description>
+
+    <!--  Suppress warning about invalid dependency version in Castle.Core.
+          That is Moq dependency and we cannot fix that somehow. -->
+    <NoWarn>$(NoWarn);NU1603</NoWarn>
   </PropertyGroup>
 
   <ItemGroup>
     <CodeAnalysisDictionary Include="CodeAnalysisDictionary.xml" />
   </ItemGroup>
 
-  <ItemGroup>
+  <ItemGroup Condition=" '$(TargetFramework)'=='net45' ">
     <PackageReference Include="Moq" Version="4.1.1308.2120" />
   </ItemGroup>
 
+  <ItemGroup Condition=" '$(TargetFramework)'=='netstandard1.5' ">
+    <PackageReference Include="Moq" Version="4.7.0" />

```

Comment:
Isn't this a breaking change?

Reply:
First of all notice that we are in v4 branch :)

However, nope, this is not a breaking change. If you install this package to .NET Framework 4.5 you will still need Moq 4.1 as a dependency requirement.

For the .NET Standard project you will need Moq 4.7, however .NET Standard support is a _new feature_, so nothing existing was broken.